Extension/src/LanguageServer/extension.ts

```diff
@@ -1062,6 +1110,37 @@ function handleMacCrashFileRead(err: NodeJS.ErrnoException | undefined | null, d
     logMacCrashTelemetry(data);
 }
 
+async function handleCrashFileRead(crashDirectory: string, crashFile: string, err: NodeJS.ErrnoException | undefined | null, data: string): Promise<void> {
```

Comment:
`crashFile` is technically correct, but when reading this function I'm thinking of the file containing the printout of the callstacks, not the name of the executable that crashed (cpptools[-srv]).  Especially because `crashDirectory` relates to the callstacks. Could you pick a different name for this?  `crashingProcess` perhaps?

Reply:
It is the name of the file containing the printout of the call stacks.

Reply:
You mentioned future support for cpptools-srv.  If that crashes really fast a few times in a row will we get unique files for each crash or do we run the risk of corruption if the file is overwritten while we're trying to read it?

Reply:
> It is the name of the file containing the printout of the call stacks, e.g. cpptools.txt

Ok, got it. I was thinking of the old telemetry that logs the process name first, I think.

Reply:
Yeah, I figured it was better to log cpptools.txt to make it easier to differentiate from cpptools-srv.